tests/test_bgutil.py

```diff
@@ -15,15 +15,30 @@
 from __future__ import print_function
 
 import unittest
+from mock import Mock, patch
+from StringIO import StringIO
 
 from biggraphite.cli import bgutil
 from biggraphite import test_utils as bg_test_utils
+from biggraphite.accessor import Metric, Retention
 
 
 class TestBgutil(bg_test_utils.TestCaseWithFakeAccessor):
 
-    def test_run(self):
-        bgutil.main(['--driver=memory', 'read', 'foo'])
+    metrics = ['metric1', 'metric2']
+
+    @patch('biggraphite.drivers.memory._MemoryAccessor.get_metric')
```

Comment:
Because I didn't want to have side effet, but ok it's better to use the memory driver. Can I reset easily the content of the memory for other tests?